lib/battle/AutocombatPreferences.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * AutocombatPreferences.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#pragma once
+
+struct AutocombatPreferences
+{
+	bool enableSpellsUsage = true;
+	//TODO: below options exist in original H3, consider usefulness of mixed human-AI combat when enabling autocombat inside battle
+	bool enableUnitsUsage = true;
+	bool enableCatapultUsage = true;
+	bool enableBallistaUsage = true;
+	bool enableFirstAidTendUsage = true;
```

Comment:
Options are not implemented. Correct? I'd rather avoid options that exists but not used - end result is usually confusing...
```suggestion
//	bool enableUnitsUsage = true;
//	bool enableCatapultUsage = true;
//	bool enableBallistaUsage = true;
//	bool enableFirstAidTendUsage = true;
```